src/handlers/resource_handler.ts

```diff
@@ -186,11 +187,11 @@ export class ResourceHandler {
       return;
     }
 
-    const clone = Drash.Prototype.clone(resource, {
-      path_parameters: pathParams,
-      request: request,
-      server: resource.server,
-    });
+    const clone = Drash.Prototype.clone(resource);
+
+    clone.path_parameters = pathParams;
+    clone.request = request;
+    clone.server = resource.server;
```

Comment:
Not sure *why* this was changed, but it's not a problem, im sure you had your reasons and im not against it, just trynna understand

Reply:
i didnt want the prototype to add the options automagically. we could probably just change this to `Object.create()` in the future